feature/experimental/p4rt/tests/p4rt_election/p4rt_election_test.go

```diff
@@ -365,9 +365,8 @@ func TestUnsetElectionid(t *testing.T) {
 				t.Fatalf("Incorrect status code received: want %d, got %d", test.wantStatus, resp)
 			}
 			if err != nil {
-				t.Errorf("Errors seen in ClientArbitration response: %v", err)
+				t.Logf("Errors seen when sending Master Arbitration as expected for unset ElectionID: %v", err)
```

Comment:
if these errors are expected, should we call t.Error if `err == nil`?

Reply:
hi @greg-dennis , this change is not going through. i will undo it.